www/templates/main-recommendations.html

```diff
@@ -9,43 +9,21 @@
      <link rel="stylesheet" type="text/css" href="slick/slick.css"/>
   <ion-content class="has-header" overflow-scroll="true" padding="false">
     <div id = "scrolling">
-     <center><div class="recommendbox" id = "0">
-       <div class = "larger_font"> Cafes and Beverage </div>
+      <center>
+    <div class="recommendbox" id = "1">
+       <div class = "larger_font"> Bubble Tea </div>
        <h4>
-
-       <div id = "message_1"> Ernie's Espresso </div>
-       <div id = "mode_1"> Walk </div>
+       {{name}}<br />
+       {{mode}}<br />
            <img src="img/small/small_4.png" width="75" />
-             <button class="button button-block button-balanced button-outline" ng-click="clickDirections(0)">Directions</button>
-             <button class="button button-block button-balanced button-outline" ng-click="clickReview(0)">Review</button>
-             <button class="button button-block button-balanced button-outline" ng-click="clickSuggestion(0)">Suggestion</button>
+             <button class="button button-block button-balanced button-outline" ng-click="clickDirections(1)">Directions</button>
```

Comment:
why do you need to change all the zeros to ones? Also isn't there only one suggestion?